modules/apps/user/user-test/src/testIntegration/java/com/liferay/user/service/test/UserLocalServiceTest.java

```diff
@@ -843,69 +1078,71 @@ public void testUnlockoutUserWithStaleLastFailedLoginDate()
 
 		User user = UserTestUtil.addUser();
 
-		PasswordPolicy passwordPolicy = user.getPasswordPolicy();
```

Comment:
Previously, you were using the user password policy, now you're using the default one. Is this correct?

Maybe you should generalize _updateDefaultPasswordPolicyWithSafeCloseable? You could create an _updatePasswordPolicyWithSafeCloseable(passwordPolicy, consumer), that takes a password policy and a consumer to update it.

Reply:
For testing purposes, it shouldn't matter if we are using the user's password policy (comapny default by default anyways) or the company default.  Unless we want to test the user-assigned password policy is taking precedence over the default password policy, we won't need to generalize _updateDefaultPasswordPolicyWithSafeCloseable.

Reply:
Sounds good. Can you resolve this conversation?